src/app/auth/auth.service.ts

```diff
@@ -179,4 +200,45 @@ var ProductName = sessionStorage.getItem('ProductName');
             return fetchresult;
         }));
     }
+
+    captchaVerify(token: string){
+
+
+ 
+
+       let httpOptions1 = {
+        headers: new HttpHeaders({
+            'Access-Control-Allow-Methods':'DELETE, POST, GET, OPTIONS',
+            'Access-Control-Allow-Headers':'Content-Type, Access-Control-Allow-Headers, Authorization, X-Requested-With',
+            'Content-Type':  'application/x-www-form-urlencoded',
+            'Access-Control-Allow-Origin':'http://localhost:4200'
+        })
+      };
+
+      this.headers1 = new HttpHeaders(
+        {
+            'Content-Type':  'application/x-www-form-urlencoded',
+            Accept: '*/*',
+
+            
+        }
+    );
+
+
+
+
+let data={
+//6LeJqqsZAAAAADgHTJGWg4YMI028ffU6c--Av2YR(client)
+//6LdoiLwZAAAAADmQpBrZnki6eFbWJS-WaD1r1luU---owner
+//secret:"6LdoiLwZAAAAADmQpBrZnki6eFbWJS-WaD1r1luU",
+
+   secret:"6LdoiLwZAAAAADmQpBrZnki6eFbWJS-WaD1r1luU",
```

Comment:
@Bithika-Redapple Do not hardcode the secret here. That is a security risk. Create a Constants class. Do not use string literals. Even better would be to have you keys in a property file.

Reply:
ok